src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -84,6 +84,9 @@
             <argument type="service" id="sylius.product_variant_resolver.default" />
         </service>
         <service id="sylius.factory.cart_item" alias="sylius.custom_factory.order_item" />
+        <service id="sylius.custom_factory.address" class="Sylius\Component\Core\Factory\AddressFactory" decorates="sylius.factory.address">
```

Comment:
We did have any convention yet, but I would vote for `sylius.factory.address.core` since `custom_factory` is too general and means nothing (I know we won't use this `id` anywhere, but still).